.rubocop.yml

```
@@ -49,3 +49,12 @@ RSpec/SpecFilePathSuffix:
 
 RSpec/VerifiedDoubleReference:
   Enabled: false
+
+RSpec/LetSetup:
+  Exclude:
+    - spec/system/placements/support/placements/add_a_placement_spec.rb
+    - spec/system/placements/schools/placements/add_a_placement_spec.rb
+
+RSpec/NoExpectationExample:
+  Exclude:
+    - spec/support/shared_examples/add_a_placement_wizard.rb
```

I think this is going to be a common problem with shared examples for system specs.

It looks like this rule is already configured to exclude system specs as part of rubocop-govuk:

https://github.com/alphagov/rubocop-govuk/blob/87565131e1e6b15569e311e78ef72260e86f475b/config/rspec.yml#L34-L39

What do you think about us creating a directory specifically for shared examples for system specs? That way we could exclude the entire directory, similar to how it's done for `spec/system` and avoid needing to update the Rubocop config every time we add shared examples for system specs.

For example, we could go with:

```suggestion
RSpec/NoExpectationExample:
  Exclude:
    - spec/support/shared_examples/system/**/*.rb
```

or if that feels too deeply nested, what about:

```suggestion
RSpec/NoExpectationExample:
  Exclude:
    - spec/support/shared_system_examples/**/*.rb
```

What do you think?